lib/LoginStrategy.ts

```diff
@@ -248,18 +249,56 @@ export class LoginStrategy extends BaseStrategy {
    * @todo use iam-lib
    */
   async offchainClaimsOf(did: string): Promise<OffchainClaim[]> {
+    did = this.didUnification(did);
+
+    const transformClaim = (claim: OffchainClaim): OffchainClaim => {
+      const transformedClaim = { ...claim };
+      const didFormatFields = ['iss', 'sub', 'subject', 'did'];
+
+      Object.keys(transformedClaim).forEach((key) => {
+        if (didFormatFields.includes(key)) {
+          transformedClaim[key] = this.didUnification(transformedClaim[key]);
+        }
+      });
+
+      return transformedClaim;
+    };
+
     if (this.cacheServerClient?.isAvailable) {
-      return this.cacheServerClient.getOffchainClaims({ did });
+      return (await this.cacheServerClient.getOffchainClaims({ did })).map(
+        transformClaim
+      );
     }
     const didDocument = await this.didResolver.read(did);
     const services = didDocument.service || [];
-    return await Promise.all(
-      services
-        .map(async ({ serviceEndpoint }) => {
+    return (
+      await Promise.all(
+        services.map(async ({ serviceEndpoint }) => {
           const claimToken = await this.ipfsStore.get(serviceEndpoint);
           return this.decodeToken<OffchainClaim>(claimToken);
         })
-        .filter(isOffchainClaim)
-    );
+      )
+    )
+      .filter(isOffchainClaim)
+      .map(transformClaim);
+  }
+
+  /**
+   * @param {string} did
+   * @returns {string} DID address in format "did:" method-name ":" method-specific-id ":" address
+   */
+  didUnification(did: string): string {
+    const didParts = did.split(':');
+
+    if (didParts.length === 4) return did;
```

Comment:
Here's what I'm thinking for using ew-did-registry. I think this would set us up to basically move the `didUnification` function into ew-did-registry/did directly. What do you think?
```suggestion
    const { foundChainInfo } = getDIDChain(did)
    if (foundChainInfo) return did;
    
    const didParts = did.split(':');
```

Reply:
Yeah, I was thinking on moving `didUnification` to `@ew-did-registry/did`. The suggestion you proposed just replace one simple line of code, could you explain me why this is a better approach?

Reply:
I agree it's not a big difference however, I think it is better because it reused the function of `getDIDChain`, which answers the question of whether or not the chainID needs to be added. I know it is a small thing but if it ever does get more complex, we can put the additional complicity in the provided method.
From a non-functional standpoint, it also makes it easier to search in the future for all of the places where we have used `getDIDChain` to have a better idea of what might break if we introduce a breaking change, for example.
Do you agree with this or not really?

Reply:
Yeah, you got me. I will update it